tests/storage_and_cookie_security_tests.rs

```diff
@@ -0,0 +1,394 @@
+use axum_gate::cookie::CookieBuilder; 
+use axum_gate::storage::memory::{MemoryAccountStorage, MemorySecretStorage};
+use axum_gate::services::{AccountInsertService, AccountStorageService, SecretStorageService, CredentialsVerifierService};
+use axum_gate::hashing::{Argon2Hasher, VerificationResult};
+use axum_gate::secrets::Secret;
+use axum_gate::utils::AccessHierarchy;
+use axum_gate::{Account, Credentials, Group, Role};
+use uuid::Uuid;
+use std::sync::Arc;
+
+/// Tests for cookie security attributes and handling
+mod cookie_security_tests {
```

Comment:
This tests should also be already done at the external [`cookie`](https://github.com/rwf2/cookie-rs) library. Remove this module if they are.